### DISCUSSION TITLE
How to load dataframe from multiple compressed files?

### DISCUSSION THREAD

**Discussion by qrilka** [nushell/nushell#10377] [Q&A]
Hello,
I was trying out nushell for working with some data. Basically we have some large files containing CSVs or JSONs. But there's a thing: they are compressed (using zstandard)  and also span multiple files. I.e. I could have a folder SOMETHING which will contain files like data_0_0_0.json.zst and data_0_0_1.json.zst and I need to create a lazy datafram to do some calculations using data in those jsons.
Is this possible with nushell? Maybe it's some constraint from polars which nushell uses?

**Comment by fdncred** [comment]
I don't think that's possible in nushell right now. If it's possible in polars, then it could be possible that someone could create a PR to support it.

**Reply by qrilka** [reply]
Thanks, I'll check what's in polars
